webapp/tests/Unit/Controller/Team/ProblemControllerTest.php

```diff
@@ -183,4 +186,42 @@ public function testInteractiveSamples(): void
         $response = $this->client->getResponse();
         self::assertEquals(404, $response->getStatusCode());
     }
+
+    /**
+     * Test that the problems page does not show page, statement, sample data before contest start.
+     */
+    public function testAccessProblemBeforeContestStarts(): void
+    {
+        $this->loadFixtures([DemoAboutToStartContestFixture::class]);
+
+        /** @var EntityManagerInterface $em */
+        $em = self::getContainer()->get(EntityManagerInterface::class);
+        $problemCount = $em->createQueryBuilder()
+            ->from(ContestProblem::class, 'cp')
+            ->select('COUNT(cp.contest)')
+            ->where('cp.contest = :contest')
+            ->setParameter('contest', 1)
+            ->getQuery()
+            ->getSingleScalarResult();
```

Comment:
Can you get the contest by its name, not by the internal number which can change.

Reply:
done